src/PresentationalComponents/Cards/OverviewDashbarCard/components/DashbarCardTagOrIcon/DashbarCardTagOrIcon.cy.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { mount } from "cypress/react";
+
+import { DashbarCardTagOrIcon } from "./DashbarCardTagOrIcon";
+import { IntlProvider } from "@redhat-cloud-services/frontend-components-translations/index";
+import messages from "../../../../../../locales/translations.json";
+import tooltipMessages from "../../../../../Messages";
+import {
+  PATHWAYS,
+  INCIDENTS,
+  IMPORTANT_RECOMMENDATIONS,
+  CRITICAL_RECOMMENDATIONS,
+} from "../../../../../AppConstants";
+
+// This component returns the appropriate Tag OR Icon component based on the title it receives
+describe("DashbarCardTagOrIcon", () => {
+  it("Should render a 'Route' icon", () => {
+    mount(<DashbarCardTagOrIcon title={PATHWAYS} />);
+
+    // ensure the 'Route' icon is displayed
+    cy.get("[data-cy=route-icon]").should("exist");
+  });
+
+  /******************************************************/
+  it("Should render 'Incidents' label", () => {
+    mount(
+      <IntlProvider locale={navigator.language.slice(0, 2)} messages={messages}>
+        <DashbarCardTagOrIcon title={INCIDENTS} />
+      </IntlProvider>
+    );
+
+    // ensure the correct text is displayed
+    cy.contains(/Incident/).trigger("focus");
+    cy.contains(tooltipMessages.incidentTooltip.defaultMessage);
+
+    // ensure that the tooltip message is hidden
+    cy.contains(/Incident/).trigger("blur");
+    cy.contains(tooltipMessages.incidentTooltip.defaultMessage).should(
+      "not.exist"
+    );
+  });
```

Comment:
All these tests, and the ones below, look all very very alike, maybe they can be refactored to share the similarities in a test helper and keep everything DYR

Reply:
The recommended way to do this in cypress is to use "commands": https://docs.cypress.io/api/cypress-api/custom-commands#Syntax

Reply:
I am working on converting all tests in this PR from Cypress to React Testing Library
Is there a way to create these test helpers in RTL ?